src/Misc/layoutbin/runsvc.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
I thought about doing entire env but I think that leads to more noise.  Since we also gave them the ability to inject stuff in the generated startup sh, they can set whatever they want